theorems/T000683.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: T000683
+if:
+  and:
+  - P000203: true
+  - P000104: true
```

Comment:
```suggestion
  - P000104: true
  - P000203: true
```
swapping the order of the hypotheses so it better matches T628 in the list of theorems for symmetrizable:
https://topology.pi-base.org/properties/P000104